vector/src/main/java/im/vector/app/features/home/room/list/RoomListViewModel.kt

```diff
@@ -242,34 +242,24 @@ class RoomListViewModel @Inject constructor(initialState: RoomListViewState,
     }
 
     private fun buildRoomSummaries(rooms: List<RoomSummary>): RoomSummaries {
-        // Set up init size on directChats and groupRooms as they are the biggest ones
```

Comment:
Sorry I just noticed this comment now. I think we should init size on `conversations` with rooms.size
Don't you think so?

Reply:
Yes it's a good suggestions for conversations list. Do you have any idea for this init value ?